app/views/support/new.html.erb

```diff
@@ -0,0 +1 @@
+<%= render SupportFormComponent.new(form:) %>
```

Comment:
Can we add a back link on this page? (It'll have to be dynamic based on which ECT, mentor other page you were looking at)

Reply:
We resolved to leave this for a follow-up PR.